lib/src/models/client-config.ts

```diff
@@ -44,6 +44,10 @@ export interface StrictAuthClientConfig {
      *
      */
     sendCookiesInRequests?: boolean;
+    /**
+     * Sets the development environment
+     */
+     devEnvironment?: string;
```

Comment:
Can we use a different name since `devEnvironment` can also refer to dev environment in the CICD flow?

Reply:
We should also add this to the readme file.

Reply:
I was thinking about `runtimeEnvironment` too but in the meeting, everyone was using `devEnvironment` so I changed this.